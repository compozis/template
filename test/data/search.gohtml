{{extends "layouts/sidebar.gohtml"}}
{{define "Sidebar"}}Search FILTERS{{end}}
{{define "MainContent"}}Search RESULTS{{end}}