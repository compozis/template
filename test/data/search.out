<html>
<head>
    <title></title>
</head>
<body>
<div class="content">
    <div class="sidebar">Search FILTERS</div>
    <div class="maincontent">Search RESULTS</div>
</div>
<div class="footer">[DEFAULT] Footer</div>
</body>
</html>
